Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -7249,8 +7261,8 @@
 			);
 			name = "Kickstarter-Framework-iOS";
 			packageProductDependencies = (
-				1998BCAD28F60EB700D04077 /* ReactiveExtensions */,
-				1905787E28F8CD5E00428375 /* ReactiveSwift */,
+				6047E6F928FDC104003A55A0 /* ReactiveExtensions */,
+				6047E6FB28FDC1B1003A55A0 /* ReactiveSwift */,
```

Comment:
Not sure why these IDs changed because they same libraries are linked. It should be ok.

Reply:
Usually ID's change when libraries are added and removed again, wouldn't worry too much about this as they generated by XCode, not us.